src/app/shared/table/table-data-source.ts

```diff
@@ -345,6 +346,10 @@ export abstract class TableDataSource<T extends TableData> {
           if (filterDef.type === 'date') {
             // Date
             filterJson[filterDef.httpId] = filterDef.currentValue.toISOString();
+          } else if (filterDef.type === FilterType.DATE_RANGE) {
+            // Date range
+            filterJson['StartDateTime'] = filterDef.currentValue.startDate.toISOString();
+            filterJson['EndDateTime'] = filterDef.currentValue.endDate.toISOString();
```

Comment:
Still the same issue I think, if I have two date time range controls, you use the same properties to build the filter!
Filter properties should be provided when the filter is built and used here with the right httpIDs (2 props)

Reply:
I have added translation within the datetime picker for the apply button. We can also translate the month names and week names within the calendar but the display on the input will always show in English, I guess. Shall I add support for translating within the calendar?
![image](https://user-images.githubusercontent.com/5779016/149310043-7d348555-a0c4-40b3-b4fb-27f5bdcdf401.png)

Reply:
Normally you can do this with moment get this info.
I this Caroline or Claude got already this done, you can check with either of them.
Thanks!